pkg/utils/cpu.go

```diff
@@ -0,0 +1,24 @@
+package utils
+
+import (
+	"syscall"
+	"time"
+)
+
+var (
+	lastInspectUnixNano int64
```

Comment:
should we need `init` `lastInspectUnixNano` when process started?

Reply:
It seems no need.   `lastCPUUsageTime` is also `0`.